processing/src/main/java/io/druid/guice/ExtensionsConfig.java

```diff
@@ -33,10 +34,6 @@
   @NotNull
   private boolean searchCurrentClassloader = true;
 
-  @JsonProperty
-  @NotNull
-  private List<String> coordinates = ImmutableList.of();
```

Comment:
> not cause any harm

Not true, if they didn't realize one of the nodes had not been updated to the "new" config, then it will not launch properly, or will have missing functionality that may not even report on bootup.